### DISCUSSION TITLE
CI: Get the repository and branch name being built

### DISCUSSION THREAD

**Discussion by reikdas** [community/community#25108] [Actions]
<p>Is there a way to find the repository and branch name being built in my Github Actions CI? If you are familiar with travis, I am looking for the alternative to travis’ TRAVIS_REPO_SLUG environment variable.</p>

**Comment by managedkaos** [comment, marked as answer]
<p>Hello, <a class="mention" href="/reikdas">@reikdas</a>!  Please consider this document from the official GitHub Actions documenation:</p>
<p><a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables" title="Using environment variables" rel="nofollow noopener">Using environment variables</a></p>
<p>It lists all the environment variables that are automatically created in the Workflow environment.</p>
<p>Among the variables are:</p>
<table><tbody><tr>
<td>GITHUB_REPOSITORY</td>
<td>The owner and repository name. For example,<span> </span>octocat/Hello-World.</td>
</tr></tbody></table>
<p>and </p>
<table><tbody><tr>
<td>GITHUB_REF</td>
<td>The branch or tag ref that triggered the workflow. For example,<span> </span>refs/heads/feature-branch-1. If neither a branch or tag is available for the event type, the variable will not exist.</td>
</tr></tbody></table>
<p>Those should cover the cases you need.</p>
<p>Please update the discussion to let readers know how you fared with this information.</p>
<p>Best,</p>
<p>ManagedKaos</p>
